infrastructure/terraform/components/acct/locals_remote_state.tf

```diff
@@ -0,0 +1,21 @@
+locals {
+  bootstrap = data.terraform_remote_state.bootstrap.outputs
+}
+
+data "terraform_remote_state" "bootstrap" {
```

Comment:
can we have this in a separate tf file please, might be easier to locate if wanted to know where its at 😅